osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/firmware/ImageTransfer.java

```diff
@@ -8,7 +8,6 @@
  */
```

Comment:
There is no unit test for this class. Adding tests for all functionality would be too much, but adding some tests for the changed functionality would be good.

Reply:
I added testing for the CommandExcuter, that also involves this class